src/neo/Network/P2P/Payloads/OracleResponse.cs

```diff
@@ -0,0 +1,73 @@
+using Neo.IO;
+using Neo.IO.Json;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract.Native;
+using Neo.SmartContract.Native.Oracle;
+using Neo.VM;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class OracleResponse : TransactionAttribute
+    {
+        private const int MaxResultSize = 1024;
+
+        public static readonly byte[] FixedScript;
+
+        public ulong Id;
```

Comment:
Who determine this ID? I think that it's better to use a hash (RequestTx.Hash,url)